spec.bs

```diff
@@ -656,6 +719,8 @@ directly; for authentication the extension can only be accessed via
         1. Skip step 6.1, which compares *options.rpId* to *effectiveDomain*
 
             Note: This enables cross-domain authentication ceremonies; see [[#sctn-use-case-merchant-authentication]].
+            
+            Q: Should we not at least compare to the claimed Bank Identity? How else will we know the value displayed is actually linked to the credential?
```

Comment:
Because currently the caller of SPC doesn't set the RP origin shown to the user. There is no input to https://w3c.github.io/secure-payment-confirmation/#sctn-securepaymentconfirmationrequest-dictionary where one can 'claim' the RP value.

How the user agent actually *gets* the RP's identity from the credential is an outstanding issue for the spec. In the Chrome impl, we cache it at creation time. In the future, I hope to retrieve it via the Conditional UI concept in WebAuthn (which will also require the ability to fetch an RP for a given credential).

We *could* ask the caller to make an assertion about what domain it is, but I think given that there are technical solutions to avoid that, we should. If we do ultimately ask the caller to make such a claim, then yes we should verify that once we know the value.